docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
-| enable_experimental_mv                   | -    | TRUE       | Whether to enable the asynchronous materialized view feature. `TRUE` indicates this feature is enabled. From v2.5.2 onwards, this feature is enabled by default. For versions earlier than v2.5.2, this feature is disabled by default. |
-| authentication_ldap_simple_bind_base_dn  | -  | Empty string | The base DN, which is the point from which the LDAP server starts to search for users' authentication information.|
-| authentication_ldap_simple_bind_root_dn  |  -  | Empty string | The administrator DN used to search for users' authentication information.|
-| authentication_ldap_simple_bind_root_pwd |  -  | Empty string | The password of the administrator used to search for users' authentication information.|
-| authentication_ldap_simple_server_host   |  -  | Empty string |  The host on which the LDAP server runs.                               |
-| authentication_ldap_simple_server_port   |  -  | 389     | The port of the LDAP server.                                       |
-| authentication_ldap_simple_user_search_attr|  -  | uid     | The name of the attribute that identifies users in LDAP objects.|
-|max_upload_task_per_be                    |  -  | 0       | In each BACKUP operation, the maximum number of upload tasks StarRocks assigned to a BE node. When this item is set to less than or equal to 0, no limit is imposed on the task number. This item is supported from v3.1.0 onwards.     |
-|max_download_task_per_be                  |  -  | 0       | In each RESTORE operation, the maximum number of download tasks StarRocks assigned to a BE node. When this item is set to less than or equal to 0, no limit is imposed on the task number. This item is supported from v3.1.0 onwards.     |
+##### plugin_enable
+
+- **Unit**: -
+- **Default**: TRUE
+- **Description**: Whether plugins can be installed on FEs. Plugins can be installed or uninstalled only on the Leader FE.
+
+##### max_small_file_number
+
+- **Unit**: -
+- **Default**: 100
+- **Description**: The maximum number of small files that can be stored on an FE directory.
+
+##### max_small_file_size_bytes
+
+- **Unit**: Byte
+- **Default**: 1024 * 1024
+- **Description**: The maximum size of a small file, in bytes.
+
+##### agent_task_resend_wait_time_ms
+
+- **Unit**: ms
+- **Default**: 5000
+- **Description**: The duration the FE must wait before it can resend an agent task. An agent task can be resent only when the gap between the task creation time and the current time exceeds the value of this parameter. This parameter is used to prevent repetitive sending of agent tasks. Unit: ms.
+
+##### backup_job_default_timeout_ms
+
+- **Unit**: ms
+- **Default**: 86400*1000
+- **Description**: The timeout duration of a backup job, in ms. If this value is exceeded, the backup job fails.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/875dd46f-fe06-4c82-bcb5-bb18770f6c15)
Delete ", in ms".